packages/account/src/Sections/Profile/PhoneVerification/resend-code-timer.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useRef } from 'react';
+import { Button, CaptionText } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+
+const ResendCodeTimer = () => {
+    const [timer, setTimer] = React.useState(60);
+    const [startTimer, setStartTimer] = React.useState(false);
```

Comment:
Since we are coding in the legacy app, best to follow the same naming conventions.
```suggestion
    const [start_timer, setStartTimer] = React.useState(false);
```

Reply:
Sure, Matin. Missed that. Got the the habit coding in V2 🥲